disclaimers.json

```diff
@@ -0,0 +1,254 @@
+[
+  {
+    "api": "cloudasset",
+    "package": "@google-cloud/asset"
+  },
+  {
+    "api": "automl",
+    "package": "@google-cloud/automl"
+  },
+  {
+    "api": "bigquery",
+    "package": "@google-cloud/bigquery"
+  },
+  {
+    "api": "bigtable",
+    "package": "@google-cloud/bigtable"
+  },
+  {
+    "api": "cloudbilling",
+    "package": "@google-cloud/billing"
+  },
+  {
+    "api": "billingbudgets",
+    "package": "@google-cloud/billing-budgets"
+  },
+  {
+    "api": "cloudbuild",
+    "package": "@google-cloud/cloudbuild"
+  },
+  {
+    "api": "datacatalog",
+    "package": "@google-cloud/datacatalog"
+  },
+  {
+    "api": "datalabeling",
+    "package": "@google-cloud/datalabeling"
+  },
+  {
+    "api": "dlp",
+    "package": "@google-cloud/dlp"
+  },
+  {
+    "api": "dataproc",
+    "package": "@google-cloud/dataproc"
+  },
+  {
+    "api": "datastore",
+    "package": "@google-cloud/datastore"
+  },
+  {
+    "api": "datastore",
+    "package": "@google-cloud/connect-datastore"
+  },
+  {
+    "api": "dialogflow",
+    "package": "dialogflow"
+  },
+  {
+    "api": "dns",
+    "package": "@google-cloud/dns"
+  },
+  {
+    "api": "documentai",
+    "package": "@google-cloud/documentai"
+  },
+  {
+    "api": "firestore",
+    "package": "@google-cloud/firestore"
+  },
+  {
+    "api": "firestore",
```

Comment:
is it a typo or do we have two instances of `firestore`?

Reply:
So this is interesting!  Technically the content here is just derived from here:
https://github.com/googleapis/google-cloud-node/blob/master/libraries.json

And those are all derived from various repo-metadata files throughout our repositories.  The issues is that we flag multiple libraries (firestore, datastore, storage, logging all come to mind) as using the underlying API.  There's no notion of "the right one".  I have no idea what to do with this problem to be honest.